sdk/appconfiguration/azure-data-appconfiguration-v2/src/main/java/com/azure/v2/data/appconfiguration/AzureAppConfigurationClient.java

```diff
@@ -55,7 +55,6 @@ class AzureAppConfigurationClient {
      * @param acceptDatetime Requests the server to respond with the state of the resource at the specified
      * time.
      * @throws IllegalArgumentException thrown if parameters fail the validation.
-     * @throws ErrorException thrown if the request is rejected by server.
```

Comment:
Was this cleanup due to a Checkstyle check flagging this or the fact that `ErrorException` isn't real.

If this was `HttpResponseException` would this have been removed?

Reply:
> the fact that ErrorException isn't real.

this, plus we have

> @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.

on the next line, which we probably should replace with `CoreException`